qiskit_ionq/ionq_client.py

```diff
@@ -186,25 +196,30 @@ def get_calibration_data(self, backend_name: str) -> dict:
         Returns:
             dict: A dictionary of an IonQ backend's calibration data.
         """
-        req_path = self.make_path("/".join([
-            "characterizations/backends",
-            backend_name[5:],
-            "current"
-        ]))
+        req_path = self.make_path(
+            "/".join(["characterizations/backends", backend_name[5:], "current"])
+        )
         res = requests.get(req_path, headers=self.api_headers, timeout=30)
         exceptions.IonQAPIError.raise_for_status(res)
 
         return res.json()
 
     @retry(exceptions=IonQRetriableError, max_delay=60, backoff=2, jitter=1)
-    def get_results(self, job_id: str, aggregation=None):
+    def get_results(
+        self,
+        job_id: str,
+        sharpen: Optional[bool] = None,
+        extra_request_payload: Optional[dict] = None,
```

Comment:
ah, sorry... when I saw that this was updating a dict I thought it was part of the jobs payload. extra_query_params does seem more appropriate (sorry to back and forth so much, this field being optional means folks will need to use its name to reference it so I'm a bit fixated on it at the moment).